docs/api/rest_api/rest_api_reference/input/examples/user/roles/id/policies/POST/PolicyCreate.xml.example

```diff
@@ -5,12 +5,12 @@
     <limitations>
         <limitation identifier="Class">
             <values>
-                <ref href="/api/ibexa/v2/content/types/13"/>
+                <ref media-type="application/vnd.ibexa.api.ref+xml" href="2"/>
```

Comment:
Oh my... This is definitely not correct, but partially on the implementation level, not the doc.

Question: do you need this `media-type`? `application/vnd.ibexa.api.ref+xml` is not a recognized type anywhere. It rather should be a type describing what `href` is, however href is not correct either. Can we for now skip media-type?

As for `href` something clearly didn't come together during the implementation. What you had before was what it should be, what you have now is what actually works... Reported as [IBX-6028](https://issues.ibexa.co/browse/IBX-6028).